src/sage/coding/linear_code.py

```diff
@@ -1553,42 +1584,44 @@ def permutation_automorphism_group(self, algorithm="partition"):
         ::
 
             sage: C = codes.GolayCode(GF(3))
-            sage: M11 = MathieuGroup(11)
-            sage: M11.order()
+            sage: M11 = MathieuGroup(11)                                                # optional - sage.groups
+            sage: M11.order()                                                           # optional - sage.groups
             7920
-            sage: G = C.permutation_automorphism_group()  # long time (6s on sage.math, 2011)
-            sage: G.is_isomorphic(M11)                    # long time
+            sage: G = C.permutation_automorphism_group()  # long time (6s on sage.math, 2011)   # optional - sage.groups
```

Comment:
May be we should update these long time statements. It should now be much less than the 6s of 2011 ;)

Reply:
I agree; comments like this make our codebase look dated.

Probably best done in a separate PR, for all such annotations at the same time.